### PR TITLE
Update foundation-day.md

### PR DESCRIPTION
**Pull request by ClaireMoles1**

### CODE REVIEW

File: service-manual/the-team/learning-and-development/foundation-day.md [hunk 1]
```diff
@@ -87,5 +87,5 @@ As preparation before the day, we ask that participants:
 
 To request a place on the foundation day, please fill in this [Civil Service Learning request form](https://civilservicelearning.civilservice.gov.uk/learning-opportunities/digital-foundation-day)
 
-Any questions? Please contact the GDS Skills Team: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:service-manager-programme@digital.cabinet-office.gov.uk)
+Any questions? Please contact: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:digital.academy@dwp.gsi.gov.uk)
```
Comment (issyl0) [inline service-manual/the-team/learning-and-development/foundation-day.md]: Thanks for this! A quick thing though: you'll need to change the text between the `[]` that currently advertises a CO address, as the actual email will go to a DWP address.

Comment (ClaireMoles1) [inline service-manual/the-team/learning-and-development/foundation-day.md]: Isobel,

Apologies, I have not done this before.

I want the emails to go to the DWP email address, so can this stay the same?

Thanks,

Claire



Claire Moles ¦ Recruitment & Skills Team ¦ Government Digital Service ¦
Cabinet Office
7th Floor, Aviation House, 125 Kingsway, London, WC2B 6NH

Tel: 07717715727

Comment (ClaireMoles1) [inline service-manual/the-team/learning-and-development/foundation-day.md]: Isobel,

I think I have just submitted the changes you requested, please let me know
if hat is not the case.

I have submitted several requests.  Apologies if anything else is not
correct - I have not done this before.  I am more than happy to sit with
you to go through the changes if needed.

Can you please advise me when all these changes will be made?

Thank you.

Kind regards,

Claire

Claire Moles ¦ Recruitment & Skills Team ¦ Government Digital Service ¦
Cabinet Office
7th Floor, Aviation House, 125 Kingsway, London, WC2B 6NH

Tel: 07717715727

Comment (issyl0) [inline service-manual/the-team/learning-and-development/foundation-day.md]: Thanks for the updates (#590). If you're around this afternoon after 3pm, I'm happy to come and explain what I meant—or at some point tomorrow! :+1:

**Comment by issyl0** [comment]
This has been superseded by #590, so I'm closing this.

**Pull request closed without merging**